setup/install/environment.md

```diff
@@ -12,7 +12,7 @@ In this step, you tell Halyard in what type of environment to install Spinnaker.
 The recommended path is a distributed installation onto a Kubernetes cluster,
 but all of these methods are supported:
 
-* [Distributed installation](#distributed-installation) on Kubernetes
+* [Distributed Kubernetes](#distributed-kubernetes) on Kubernetes
```

Comment:
This should work, but it failed the rake test. Can you take a look, Andres?

Reply:
i think it might be related to the change in the title. Something in the navegation broke. Is a very small change. I will not change the link only the tile lets see.